lib/resources/users.js

```diff
@@ -54,8 +54,11 @@ module.exports = (service, endpoint) => {
             .then((token) => mail(savedUser.email, 'accountCreated', { token })))
           .then(always(savedUser)))));
 
+    const endpointByInvalidateQuery = (cb) => (req, ...rest) =>
```

Comment:
`endpointByInvalidateQuery` and `endpointByTestPath` above make me think that if/when we update `endpoint` to accept an options object, we should make the `anonymous` option take a callback.

Maybe it'd be reasonable to do something like that now? What if `anonymousEndpoint` accepted a function returning `true`/`false`, and then `anonymousEndpoint` either called `authHandler` or not based on the result? We could even make the function required, since Ramda `F` would be easy to pass in.

Just a thought, I'm also happy with the approach here. :+1: